cpp/ql/lib/semmle/code/cpp/controlflow/IRGuards.qll

```diff
@@ -249,6 +289,18 @@ private class GuardConditionFromIR extends GuardCondition {
     )
   }
 
+  /**
+   * Holds if (determined by this guard) `e < k` must be `isLessThan` in `block`.
+   * If `isLessThan = false` then this implies `e >= k`.
+   */
```

Comment:
Similarly redundant?

Reply:
Fixed in 3a7b80da47417183f73fe68b67ba0c647ab65c8f